egs2/ofuton_p_utagoe_db/svs1/local/check_align.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
```

Comment:
Can we make this file generalized? (e.g., in `pyscripts`)

Reply:
For now, it differs among different corpus (some don't need).  It contains customed syllabel-to-phone translation for alignment. So maybe we can move it to pyscripts/utils.

Reply:
Since we already have a general version, can we remove this?